src/pyobs/aop.py

```diff
@@ -155,6 +155,9 @@ def __init__ (self,aerFiles,config=None,mieRootDir=None,band=False,verbose=False
 
         if config is None:
             config = G2G_MieMap
+        elif type(config) is str:
```

Comment:
Is this needed? The --config command line option worked for me without this change.

Reply:
yes, this is needed if you're using this class directly, not running from the command line.